.github/workflows/dependacy_merge.yml

```
@@ -0,0 +1,81 @@
+name: Dependabot fetch metadata
+on: pull_request
+
+permissions:
+  pull-requests: write
+  issues: write
+  repository-projects: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.user.login == 'dependabot[bot]'
+    steps:
+      - name: Dependabot metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@v1
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Find the tests for the dependency requirement
+        id: yaml
+        uses: mikefarah/yq@master
+        with:
+          cmd: yq e ".${{ steps.metadata.outputs.dependency-names }}" ./.github/tests.yaml
```

```suggestion
          cmd: yq e ".${{ steps.metadata.outputs.dependency-names }}" ./.github/dependency_tests.yaml
```
I would rename the tests yaml